releaseNotes/AzureDevOpsPipelineExtension.md

```diff
@@ -5,7 +5,10 @@ The Azure App Configuration extension in Azure DevOps pipeline has been publishe
 ![sample](pictures/AzureDevOpsExtensionVersionSample.PNG)
 
 ### v6.0.0 - February, 02 2023
-* Updated the task to require Node.js 16. It previously required 10.
+* **Breaking changes**:
```

Comment:
Breaking changes shouldn't be a bullet, it should be a header.

Reply:
Fixed breaking change header